.github/workflows/build-test.yml

```
@@ -20,12 +20,23 @@ jobs:
     runs-on:                    ${{ matrix.platform }}
     steps:
       - name:                   Checkout sources
-        uses:                   actions/checkout@v2
+        uses:                   actions/checkout@master
         with:
-          fetch-depth:          50
-      - name:                   Checkout submodules
-        shell:                  bash
-        run:                    git submodule update --init --recursive
+          submodules:           true
+      # https://github.com/actions/cache/issues/133 
+      - name:                   Fixup the owner of ~/.cargo/
+        if:                     matrix.platform == 'ubuntu-16.04'
+        run:                    sudo chown -R $(whoami):$(id -ng) ~/.cargo/
+      - name:                   Fixup the owner of C:\Users\runneradmin\.cargo\registry
+        if:                     matrix.platform == 'windows-latest'
+        shell:                  powershell
+        run:                    |
+          $registry-path = "C:\Users\runneradmin\.cargo\registry"
```

```
Unexpected token '-path' in expression or statement.
```

```suggestion
          $registry_path = "C:\Users\runneradmin\.cargo\registry"
```

I assume `_` is allowed.